src/types/user.go

```diff
@@ -0,0 +1,6 @@
+package types
+
+type User struct {
+	Username string `json:"username"`
```

Comment:
Are you considering it as required parameters? If yes, please make it as:
```
Username string `json:"username" binding:"required"`
```
Also password below.